cypress/support/pages/cart.pageObject.js

```diff
@@ -0,0 +1,81 @@
+import PageObject from '../PageObject';
+
+class CartPageObject extends PageObject {
+  url = '/cart.html';
+
+  get nameField() {
+    return cy.get('#name');
+  }
+
+  get countryField() {
+    return cy.get('#country');
+  }
+
+  get cityField() {
+    return cy.get('#city');
+  }
+
+  get cardField() {
+    return cy.get('#card');
+  }
+
+  get monthField() {
+    return cy.get('#month');
+  }
+
+  get yearField() {
+    return cy.get('#year');
+  }
+
+  typeMessage(message) {
+    this.messageField.type(message);
+  }
+
+  typeName(name) {
+    this.nameField.type(name, { force: true });
+  }
+
+  typeCountryField(country) {
+    this.countryField.type(country, { force: true });
+  }
+
+  typecityField(city) {
+    this.cityField.type(city, { force: true });
+  }
```

Comment:
The method `typecityField` should be renamed to `typeCityField` to maintain consistent camelCase naming convention with other methods like `typeCountryField` and `typeNameField`.